src/main/java/cart/error/ApiControllerHandler.java

```diff
@@ -0,0 +1,53 @@
+package cart.error;
+
+import cart.error.exception.CartException;
+import cart.error.exception.ErrorCode;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.BindException;
+import org.springframework.validation.FieldError;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@ControllerAdvice
+public class ApiControllerHandler {
+
+    private final Logger log = LoggerFactory.getLogger(getClass());
+
+    @ExceptionHandler(CartException.class)
+    public ResponseEntity<ErrorResponse> handlingApplicationException(final CartException e) {
+        final ErrorCode errorCode = e.getErrorCode();
+        log.error(
+                "{\n" +
+                        "\n\t\"status\": " + errorCode.getStatus() + '\"' +
+                        ",\n\t\"code\": \"" + errorCode.getCode() + '\"' +
+                        ",\n\t\"message\": \"" + errorCode.getMessage() + '\"' +
+                        "\n}"
+        );
```

Comment:
개행문자 등을 포함해서 사용하는 것은 os마다도 조금 다르게 동작할 수도 있고, 로그표현도 읽기 좋을까하는 생각이 듭니다.
```suggestion
        log.error("status:{}, code:{}", errorCode.getStatus(), errorCode.getCode());
```
예시처럼 표현해 보는건 어떨까요?